packages/stencil/src/components/datacom-radio/datacom-radio.tsx

```diff
@@ -55,14 +58,21 @@ export class DatacomRadio implements FormControl {
   private inputElement: HTMLInputElement;
 
   handleChange = () => {
+    console.log('handlechange', this.checked);
```

Comment:
Shouldn't leave console logs in final code (unless in some dire debugging situations)